.gitignore

```diff
@@ -15,3 +15,4 @@ config.json
 vendor/re2/obj/libre2.a
 /bazel-*
 compile_commands.json
+.devbox
```

Comment:
I'm happy to take this, but fyi you can also use `.git/info/exclude` to get un-versioned local excludes, or set `core.excludesfile` in `~/.gitconfig` to populate a user-local exclude.